ios/FluentUI/Core/Theme/Tokens/ControlTokenSet.swift

```diff
@@ -178,8 +178,8 @@ public class ControlTokenSet<T: TokenSetKey>: ObservableObject {
 /// Union-type enumeration of all possible token values to be stored by a `ControlTokenSet`.
 public enum ControlTokenValue {
     case float(() -> CGFloat)
-    case dynamicColor(() -> DynamicColor)
-    case fontInfo(() -> FontInfo)
+    case color(() -> UIColor)
+    case font(() -> UIFont)
```

Comment:
Should these names have `ui` at the front? So it's not confusing if people try to use them in SwifUI and expect the actual `color`? Oh, but I guess we already to `float` not `cgFloat`, so it's probably fine.

Reply:
Fair point - I'm inclined to keep the API name concise, but if you think it'll be especially confusing then I'm open to changing it.

Reply:
Since FUA is largely UIKit-based (based on my understanding), I personally don't think adding `ui` to the front is necessary.